core/templates/dev/head/components/private_info_tooltip/private_info_tooltip_directive.html

```diff
@@ -0,0 +1,37 @@
+<style>
+  private-info-tooltip .conversation-skin-tutor-card-tooltip {
+    display: inline-block;
+    padding-top: 5px;
+  }
+
+  private-info-tooltip .conversation-skin-tutor-card-tooltip-button {
+    background-color: #d3d3d3;
+    border-radius: 50%;
+    color: #ffffff;
+    font-size: 12px;
+    font-weight: 900;
+    padding-top: 4px;
+    position: relative;
+    text-align: center;
+  }
+
+  private-info-tooltip .conversation-skin-tutor-card-tooltip-text {
+    background-color: #A9A9A9;
+    border-radius: 3%;
+    color: #ffffff;
+    padding: 2px;
+    position: absolute;
+    visibility: hidden;
+    width: 400px;
+  }
+
+  private-info-tooltip .conversation-skin-tutor-card-tooltip-button:hover .conversation-skin-tutor-card-tooltip-text {
+    visibility: visible;
+  }
+</style>
+
+<div class="conversation-skin-tutor-card-tooltip">
+  <button class="conversation-skin-tutor-card-tooltip-button">?
```

Comment:
Do we need a button for this as this help icon is just having a tooltip (this button doesn't do anything at all)?

Reply:
Yes, this button is responding on hovering but not on clicking.

Reply:
A button is something which should do some work on click. When a user comes in contact with this button they will assume that it will do something after a click but it'll not and they will assume either there is some internet issue or a bug. Also, On A11y (voice-over) this will say it as a button.. isn't it? You can check the below link for an example:
https://demo.tutorialzine.com/2014/07/css-inline-help-tips/

@tjian11, do we need a button for this? (any suggestion?)